doc/chips/10.rst

```diff
@@ -340,14 +341,14 @@ initialization:
 	Child Phase 2
 
 
-The syntax to call a sibling initializer is ``this.init(<args>);``.  Similarly
-to in the parent-referential case, the return from that call will indicate that
-the calling initializer has begun Phase 2. In contrast to the parent case,
-however, a call to a sibling initializer may not be made if the current
-initializer has defined any fields.  The rationale for this decision is that a
-valid sibling initializer will initialize all of its fields, and so any
-initialization prior to that call will lead to the occurrence of an unexpected
-double initialization.
+The syntax to call another initializer defined on the same type is
+``this.init(<args>);``.  Similarly to in the parent-referential case, the return
+from that call will indicate that the calling initializer has begun Phase 2. In
+contrast to the parent case, however, a call to an outside initializer may not
+be made if the current initializer has defined any fields.  The rationale for
```

Comment:
It was this way before, but "define a field" is a compiler-ism. I'd probably call say "if the current initializer includes field initialization statements in Phase 1"